src/components/StaffPayment/StaffPayment.stories.ts

```diff
@@ -0,0 +1,27 @@
+import { StaffPayment } from './index'
+import { StaffPaymentOptions } from '@/enums'
+
+export default {
+  title: 'component/StaffPayment',
+  component: StaffPayment,
+  argTypes: {
+  }
+}
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { StaffPayment },
+  template: '<staff-payment v-bind="$props" />'
+})
+
+export const Default = Template.bind({})
+Default.args = {
+  staffPaymentData: {
```

Comment:
Would it be good to specify the interface here? Eg,
`Default.args: StaffPaymentIF = { ... }`

Reply:
Will do the change

Reply:
Cool.
It probably doesn't help much for Storybook, but it makes it explicit if a developer copies this code.